examples/annotation/flows/simple_annotation/flow.yaml

```
@@ -0,0 +1,41 @@
+    block_name: simple_annotation
+    input_cols:
+    - annotation_prompt
+    output_cols:
+    - raw_output
+    model: hosted_vllm/meta-llama/Llama-3.3-70B-Instruct
+    api_base: http://localhost:8000/v1
+    temperature: 0.0
+    max_tokens: 5
+    extra_body:
+      guided_choice:
+        - World
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)